<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>640</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 1</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="accessibleName">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Введите координату x:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="labelY0">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>tmp</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditX">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="8">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>138</width>
        <height>320</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Введите координату y:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEditY">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="addButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Добавить точку</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="delButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Удалить точку</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="delAllButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Удалить все точки</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="solveButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Получить ответ</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="discardButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>108</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="aboutButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>О программе</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="authorButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Об авторе</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" rowspan="2">
     <widget class="QPushButton" name="outButton">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>41</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>537</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="4">
     <widget class="QLabel" name="labelY1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>tmp</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="labelX0">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>tmp</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QLabel" name="labelX1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="text">
       <string>tmp</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
